### ISSUE TITLE
Incorrect Type-Only Import of HttpError

### ISSUE THREAD

**Issue by stateoflux** [Psifi-Solutions/csrf-csrf#13]
Hello,

I'm getting the following error when running jest:

```
node_modules/.pnpm/csrf-csrf@2.2.2/node_modules/csrf-csrf/lib/index.d.ts:2:15 - error TS1005: ',' expected.

2 import { type HttpError } from "http-errors";
                ~~~~~~~~~
```

I believe that your syntax for importing the `HttpError` type is incorrect (here is the line in [src](https://github.com/Psifi-Solutions/csrf-csrf/blob/main/src/index.ts#L5)) and should be:

```
import createHttpError from "http-errors";
import type { HttpError } from "http-errors";
```

**Comment by psibean** [comment]
What version of TypeScript are you using?

Are you getting a compilation error, or a linting error?

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md#-with-inline-type-imports

See here that it is a valid syntax, I believe from TypeScript 4.2 onwards

**Comment by stateoflux** [comment]
I'm on version 3.9.7.  I think changing the statement such that its more explicit is a good idea.  Should I create a PR for this?

**Comment by psibean** [comment]
Yeah, I'd be fine to see the type only import on it's own explicit line, and using the 3.8 syntax.

Maybe we should update the README to specify the minimum TypeScript requirement >= 3.8 too.

Maybe

`<p>Requires TypeScript >= 3.8</p>`

Before the npm install sample under the Getting Started section

**Comment by stateoflux** [comment]
done.  thx for the quick response.

**Comment by psibean** [comment]
Merged, you can point to master in the mean time, but I'll get a patch release out for it a bit later.

**Comment by stateoflux** [comment]
great! thx

**Comment by psibean** [comment]
Fixed and release 2.2.3
